src/uu/od/src/od.rs

```diff
@@ -88,85 +91,265 @@ Any type specification can have a "z" suffix, which will add a ASCII dump at
-        "BYTES",
-    );
-    opts.optopt(
-        "N",
-        "read-bytes",
-        "limit dump to BYTES input bytes",
-        "BYTES",
-    );
-    opts.optopt(
-        "",
-        "endian",
-        "byte order to use for multi-byte formats",
-        "big|little",
-    );
-    opts.optopt(
-        "S",
-        "strings",
-        "output strings of at least BYTES graphic chars. 3 is assumed when \
-         BYTES is not specified.",
-        "BYTES",
-    );
-    opts.optflagmulti("a", "", "named characters, ignoring high-order bit");
-    opts.optflagmulti("b", "", "octal bytes");
-    opts.optflagmulti("c", "", "ASCII characters or backslash escapes");
-    opts.optflagmulti("d", "", "unsigned decimal 2-byte units");
-    opts.optflagmulti("D", "", "unsigned decimal 4-byte units");
-    opts.optflagmulti("o", "", "octal 2-byte units");
-
-    opts.optflagmulti("I", "", "decimal 8-byte units");
-    opts.optflagmulti("L", "", "decimal 8-byte units");
-    opts.optflagmulti("i", "", "decimal 4-byte units");
-    opts.optflagmulti("l", "", "decimal 8-byte units");
-    opts.optflagmulti("x", "", "hexadecimal 2-byte units");
-    opts.optflagmulti("h", "", "hexadecimal 2-byte units");
-
-    opts.optflagmulti("O", "", "octal 4-byte units");
-    opts.optflagmulti("s", "", "decimal 2-byte units");
-    opts.optflagmulti("X", "", "hexadecimal 4-byte units");
-    opts.optflagmulti("H", "", "hexadecimal 4-byte units");
-
-    opts.optflagmulti("e", "", "floating point double precision (64-bit) units");
-    opts.optflagmulti("f", "", "floating point single precision (32-bit) units");
-    opts.optflagmulti("F", "", "floating point double precision (64-bit) units");
-
-    opts.optmulti("t", "format", "select output format or formats", "TYPE");
-    opts.optflag(
-        "v",
-        "output-duplicates",
-        "do not use * to mark line suppression",
-    );
-    opts.optflagopt(
-        "w",
-        "width",
-        "output BYTES bytes per output line. 32 is implied when BYTES is not \
-         specified.",
-        "BYTES",
-    );
-    opts.optflag("", "help", "display this help and exit.");
-    opts.optflag("", "version", "output version information and exit.");
-    opts.optflag(
-        "",
-        "traditional",
-        "compatibility mode with one input, offset and label.",
-    );
+pub(crate) mod options {
+    pub const ADDRESS_RADIX: &str = "address-radix";
+    pub const SKIP_BYTES: &str = "skip-bytes";
+    pub const READ_BYTES: &str = "read-bytes";
+    pub const ENDIAN: &str = "endian";
+    pub const STRINGS: &str = "strings";
+    pub const FORMAT: &str = "format";
+    pub const OUTPUT_DUPLICATES: &str = "output-duplicates";
+    pub const TRADITIONAL: &str = "traditional";
+    pub const WIDTH: &str = "width";
+    pub const HELP: &str = "help";
+    pub const VERSION: &str = "version";
+    pub const FILENAME: &str = "FILENAME";
+}
 
-    opts
+fn create_opts() -> clap::App<'static, 'static> {
```

Comment:
please move that into uumain to keep the consistency with other programs

Reply:
Done